src/test/java/com/auth0/client/mgmt/ResourceServerEntityTest.java

```diff
@@ -0,0 +1,76 @@
+package com.auth0.client.mgmt;
+
+import com.auth0.json.mgmt.ResourceServer;
+import com.auth0.json.mgmt.Scope;
+import com.auth0.net.Request;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import static com.auth0.client.MockServer.MGMT_CONNECTION;
+import static com.auth0.client.MockServer.MGMT_RESOURCE_SERVERS_LIST;
+import static com.auth0.client.MockServer.bodyFromRequest;
+import static com.auth0.client.RecordedRequestMatcher.hasHeader;
+import static com.auth0.client.RecordedRequestMatcher.hasMethodAndPath;
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+public class ResourceServerEntityTest extends BaseMgmtEntityTest {
```

Comment:
More tests should be added to cover all the new methods.

Reply:
done